notebooks/VectorSearch_QuestionRetrieval.ipynb

```diff
@@ -0,0 +1,254 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    pq_index_mem = pq_index.pq_dim * pq_index.size * pq_index.pq_bits
</pre><p>Just for readability, can we put a space after the first <code>print</code> as well? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/raft/pull/1662/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>